src/index.js

```diff
@@ -22,12 +22,21 @@ const injectDOM = function CozyBarInjectDOM ({lang, appName, iconPath}) {
   const appNode = document.querySelector('[role=application]')
```

Comment:
Same as previous comment : what if '[role=application]' element does not exist ?